CoEpi/ui/healthquiz/QuestionCell.swift

```diff
@@ -14,6 +15,17 @@ class QuestionCell: UITableViewCell {
             this.onChecked?(q, checked)
         }
         accessoryView = checkBox
+        guard let customFont = UIFont(name: "Roboto-Regular", size: 18) else {
```

Comment:
We should put the fonts in a central place (`Fonts` or however you want to name it), to not have to repeat this code and cache them.

Reply:
Added CoEpi/ui/resources/fonts/Fonts.swift